frontend/tailwind.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
```

Comment:
Hattest du den neuen JIT mode schon getestet? Können wir den schon als Default nutzen, oder spricht etwas dagegen? Ist ja jetzt zumindest im core, wenn auch noch als opt-in.

Reply:
I actually tried to keep it simple at first, but you are right, I also was curious and turned it on now 😄

It is in a preview mode. But as turning it off and on is done with a single line in the config, we can easily turn it back off, if we detect any issues.